mmf/modules/hf_layers.py

```diff
@@ -31,6 +30,7 @@ def replace_with_jit():
     BertSelfAttention.transpose_for_scores = BertSelfAttentionJit.transpose_for_scores
     BertModel.forward = BertModelJit.forward
     PreTrainedModel.__ignored_properties__ = ["base_model", "dummy_inputs"]
+    RobertaEmbeddings.forward = RobertaEmbeddingsJit.forward
```

Comment:
As we have seen this monkey patching inside helpful internally, we need to find a way to patch embeddings through backend maybe.

Reply:
The `hf_layers` contains all the Huggingface layers only and `replace_with_jit` is called from that backend. Once we add the MMF Transformer to Fluent2 we can make the change as required.